src/python/WMCore/WMSpec/Steps/Executors/LogCollect.py

```diff
@@ -63,18 +64,24 @@ def execute(self, emulator=None):
         overrideCatalog = getattr(self.step.application, 'overrideCatalog', None)
 
         overrides = {}
+        params = {}
         if hasattr(self.step, 'override'):
             overrides = self.step.override.dictionary_()
+            if 'siteLocalConfig' in overrides.keys():
+                self.siteCfg = loadSiteLocalConfig()
+                params = self.siteCfg.localStageOut
+            else:
+                params = overrides
 
         # Set wait to over an hour
         waitTime = overrides.get('waitTime', 3600 + (self.step.retryDelay * self.step.retryCount))
 
         # hardcode CERN EOS T2_CH_CERN stageout parameters
         eosStageOutParams = {}
-        eosStageOutParams['command'] = overrides.get('command', "xrdcp")
-        eosStageOutParams['option'] = overrides.get('option', "--wma-disablewriterecovery")
-        eosStageOutParams['phedex-node'] = overrides.get('phedex-node', "T2_CH_CERN")
-        eosStageOutParams['lfn-prefix'] = overrides.get('lfn-prefix', "root://eoscms.cern.ch//eos/cms")
+        eosStageOutParams['command'] = params.get('command', "xrdcp")
```

Comment:
and in the light of  those changes  this variable  name is no longer relevant, since once we merge this  PR `eos` is not going to be the sole destination for logcollect

Reply:
a better name would be: `logtStageoutParams` @amaltaro what do you think here?

Reply:
and this would also require some additional changed in the component's structure further down: look at the comments bellow